src/lib/config/nav-menu/footer-nav-items.ts

```diff
@@ -60,7 +60,7 @@ export const footerNavItems = {
                 navItems.support,
             ]
         },
-        navItems.careers,
+        // navItems.careers,
```

Comment:
Are we expecting to need to uncomment this in the future?

Reply:
@testflyjets yes, Oanh said this needs removed only this release because it won't be ready, but we'll need it in the next one.